TestHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -95,7 +95,7 @@ public void MockDirectory_GetFiles_ShouldFilterByExtensionBasedSearchPatternWith
             var fileSystem = SetupFileSystem();
             fileSystem.AddFile(additionalFilePath, new MockFileData(string.Empty));
             fileSystem.AddFile(XFS.Path(@"c:\a\a\c.gifx.xyz"), new MockFileData(string.Empty));
-            fileSystem.AddFile(XFS.Path(@"c:\a\a\c.gifx\xyz"), new MockFileData(string.Empty));
+            fileSystem.AddFile(XFS.Path(@"c:\a\a\c.gifz\xyz"), new MockFileData(string.Empty));
```

Comment:
Why did we change the test data for this?

Reply:
That test depended on the ability to have a file and directory with the same name. 9fad978a0a4d185764000351467e60a37bfd55b1